### DISCUSSION TITLE
How do I feed the config file into the system?

### DISCUSSION THREAD

**Discussion by cameronj86** [adrium/knightadjuster#5] [Q&A] [answered]
I have it saved in the same location as the ReadMe but it isn't being trigger via the Night Light, and for some reason, I can't trigger it via the terminal as shown in the video example. My uneducated guess is that my config file is not being fed into the system perhaps?

Here are my [Night Color settings](https://i.imgur.com/V8fwK4U.png) and here are my [config/plasma style settings](https://i.imgur.com/YyOVJtD.png).

**Edit**: Should it be knightadjuste**r.rc** instead of knightadjuste**rrc** perhaps?

**Comment by adrium** [comment, marked as answer]
Night Light does not trigger the knightadjuster script.

Instead, the script uses the Night Light temperature to decide, if dark or light mode should be enabled. (Btw, this is to make it an even more lightweight alternative to justjokiing/kshift)

Place the script somewhere to into your PATH to run it from the console (or use `./knightadjuster`)

I put mine in `~/.local/bin` and set up `crontab -e` to run it every 10 minutes

**Comment by cameronj86** [comment]
Stumbled across this randomly this week. This question was literally my first week using Linux as my daily driver, so I didn't know what the hell was going on even with your response lol.

Since then I've gotten a bit more experience (even wrote a night mode script that queried wttr.in, albeit not dependable at all) and stumbled back on this page.

The integration w/ night color is pretty ingenious and as lightweight as it gets. Thanks again for sharing it publicly. The one piece of feedback I'd have is to include the steps of making the file an executable in the readme for newbies. Revisiting the thread w/ more experience, I realize now that this was the step I was missing and why my files never triggered.
